Build tests only when requested.
--- a/miracle-wm-c/CMakeLists.txt
+++ b/miracle-wm-c/CMakeLists.txt
@@ -5,7 +5,9 @@ pkg_check_modules(MIRAL miral REQUIRED)
 pkg_check_modules(YAML REQUIRED IMPORTED_TARGET yaml-cpp)
 pkg_check_modules(XKBCOMMON REQUIRED xkbcommon)
 pkg_check_modules(GLIB REQUIRED IMPORTED_TARGET glib-2.0)
+if(ENABLE_TESTS)
 find_package(GTest REQUIRED)
+endif()
 
 add_library(miracle-wm-c
     SHARED
@@ -62,6 +64,7 @@ install(
 )
 
 # Tests
+if(ENABLE_TESTS)
 add_executable(test_miracle_wm_c_api
     tests/test_config_c.cpp
     tests/test_config_cpp.cpp
@@ -86,3 +89,4 @@ target_link_libraries(test_miracle_wm_c_api
 )
 
 add_test(NAME test_miracle_wm_c_api COMMAND test_miracle_wm_c_api)
+endif()
